PIL/Image.py

```diff
@@ -1209,6 +1278,7 @@ def getextrema(self):
         return self.im.getextrema()
 
     def getim(self):
+        # type: () -> Any
```

Comment:
I couldn't find an explicit type for these so far. This may be an issue for mypy or typeshed, or perhaps we just leave it at `Any` for now?